downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -113,19 +113,79 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
     ```
 
 #### OpenAPI tool
-
+-  Added support to generate a mock client for OAS if including the example details. To generate mock client, use `--mock` flag along with openAPI CLI client generation command.
+  ```
+  $ bal openapi -i <yml file> --mode client --mock
+  ```
+
+-  Provided the option to generate a single bal file for client/service. To generate single file, use `--single-file` flag along with openAPI CLI client/service generation command.
+  ```
+  $ bal openapi -i <yml file> --mode <client/service> --single-file
+  ```
+
+-  Added support to generate Ballerina service object type from given OAS. To generate a service contract object type file, use the `--with-service-contract` flag along with the OpenAPI CLI service generation command.
```

Comment:
```suggestion
-  Added support to generate the Ballerina service contract object type for a given OAS. To generate the service contract object type, use the `--with-service-contract` flag along with the OpenAPI CLI client/service generation command.
```

Reply:
@MaryamZi , I'm a bit confused about using the keywords `flag` and `option`. If we require additional information or arguments to specify a value or a set of values, we use options. and if we only provide boolean switches that turn on or off a certain feature or behavior, we mention them as flags. In our use case, we have boolean switches rather than providing values. Please correct me if I have misunderstood this.

Reply:
Yeah, I guess it's okay then. But, we generally stick to option. E.g., help text.